modules/generic_worker/templates/generic-worker.config.erb

```diff
@@ -20,7 +20,7 @@
   "privateIP": "",
   "provisionerId": "releng-hardware",
   "publicIP": "<%= @ipaddress %>",
-  "refreshURLsPrematurelySecs": 310,
+  "rootURL": "https://taskcluster.net",
```

Comment:
nit: it would be nicer to preserve alphabetical ordering, and put `rootURL` after `requiredDiskSpaceMegabytes`.